### DISCUSSION TITLE
Control Focus Through Native SDK ACAP 4

### DISCUSSION THREAD

**Discussion by whittenator** [AxisCommunications/acap-native-sdk-examples#155] [Q&A] [answered]
Is there a way to control the PTZ and/or focus of the camera through the Native SDK ACAP 4 version? I am trying to implement a custom auto-focusing algorithm and I will need to be able to control the focus.

**Comment by marbali8** [comment, marked as answer]
Hello!

Sorry for the delay. We unfortunately don't have a PTZ API in ACAP4 yet.

**Reply by whittenator** [reply]
Thanks for the reply! Any roadmap or timeline for that library into ACAP4 or would I be better off transitioning back to ACAP3?

**Reply by marbali8** [reply]
We are currently trying to close the gap between ACAP3 and ACAP4, which also means including some API for PTZ in (at least) the ACAP Native SDK. However, we can't give an estimation of when that will be.
